docs/Encapsulation.md

```diff
@@ -47,9 +47,9 @@ fastify.register(async function authenticatedContext (childServer) {
   childServer.register(require('fastify-bearer-auth'), { keys: ['abc123'] })
 
   childServer.route({
-    path: '/one',
+    url: '/one',
```

Comment:
`path` is a valid alias.

Reply:
My bad, so seems like typing is wrong: https://github.com/fastify/fastify/blob/master/types/route.d.ts#L111